### DISCUSSION TITLE
Samsung Internet (Chrome 83) missing polyfill

### DISCUSSION THREAD

**Discussion by kllmanu** [vercel/next.js#22414] [Help]
I need to support Samsung Internet browser on Android. It is setup as the default browser on most Samsung smartphones including all the Galaxy flagships, even if you've Google Chrome installed. If people open links shared with Whatsapp, it will open Samsung Internet browser. So a lot of people who are not proficient with tech, will have issues with the following:

- Samsung Internet uses Chrome 83, which does not support `String.replaceAll` and on vercel it will end up with a white page saying `An unexpected error occured`
- I've tried to add Chrome 83 in the targets with no success so far, it seems to have no effect and `String.replaceAll` is still missing as a polyfill

```
module.exports = {
  presets: [
    [
      "next/babel",
      {
        "preset-env": {
          targets: {
            chrome: "83",
          },
        },
        "preset-react": { runtime: "automatic" },
      },
    ],
  ],
  plugins: ["babel-plugin-macros", ["styled-components", { ssr: true }]],
};

```

Any ideas on what I could try next?

**Comment by tomsvogel** [comment]
easy solution is not to use replaceAll 

``` 
const newTextVar = textVar.replace(
        new RegExp('text to replace', 'g'),
        'new text'
      );
```
